### ISSUE TITLE
Feature request: Display trailing whitespace in error output

### ISSUE THREAD

**Issue by Stretch96** [wata727/tflint#216]
Currently when there is trailing whitespace, it errors with something along the lines of:

```
diff a/soemthing.tf b/something.tf
[...]
- a_thing = "thing" 
+ a_thing = "thing"
```

It's not clear that it's a trailing whitespace error, without opening a file in an editor that shows trailing whitespaces

I'm not sure how this could be displayed to the user, perhaps:

```
diff a/soemthing.tf b/something.tf
[...]
- a_thing = "thing" <<< Trailing Whitespace
+ a_thing = "thing"
```

**Comment by wata727** [comment]
I think `terraform fmt` is more suitable than TFLint in this case. Such a style problem should be automatically corrected. What do you think?

**Comment by Stretch96** [comment]
Ah, I just noticed that was the output from `terraform fmt -check -diff`, not `tflint` 🙈
(we run both in testing)

Apologies ...

Though it does raise the issue whether `tflint` should raise warnings for trailing white space, which currently it doesn't.

I feel `terraform fmt` should only be used to quickly fix errors before a commit is made, thereby fixing the errors raised by `terraform fmt -check -diff`, which can be ran in CI/CD tests. (Although I just manually fix the errors it raises rather than running `terraform fmt`).

Also I don't believe any code changes should be made in CI/CD, except things like compilation, so I don't think `terraform fmt` would be a good replacement to not checking for certain style issues.
